control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -967,3 +966,23 @@ func buildKonnectivityVolumeClusterCerts(v *corev1.Volume) {
 		DefaultMode: pointer.Int32(0640),
 	}
 }
+
+func RenderAuditLogScript(auditLogFilePath string) string {
+	var script = `
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function cleanup() {
+	for child in $( jobs -p ); do
+		kill "${child}"
+	done
+	wait
+}
+trap cleanup EXIT
```

Comment:
I tested this and it doesn't work unless `EXIT` is `SIGTERM`.

Reply:
@rtheis I tested this using the openshift-apiserver
```
[prestage-dal10-carrier100] joseph.goergen@prestage-mon01-carrier1-worker-1017:~$ time kubectl delete pod -n master-coskjif10c6fhqphi0hg openshift-apiserver-55476ddfc-h4ldv
pod "openshift-apiserver-55476ddfc-h4ldv" deleted

real	1m47.992s
user	0m0.273s
sys	0m0.109s
```

Reply:
@Joseph-Goergen my point exactly, it shouldn't take 1 minute 47 seconds to fully terminate the pod.